cl/opinion_page/forms.py

```diff
@@ -250,141 +193,236 @@ def __init__(self, *args, **kwargs) -> None:
         self.initial["court_str"] = self.pk
         self.initial["court"] = Court.objects.get(pk=self.pk)
 
-        if self.pk == "me":
-            # The court requested the order of the panel match the seniority
-            # of the judges, in order or date joined after sorting by pos type
-            # Chief, Associate, Retired Active
-            # Additionally, we only want active justices so remove them if
-            # terminated or retired, without a new role being created as an
-            # retired active justice
-            q_judges = (
-                Person.objects.filter(
-                    (
-                        (
-                            Q(positions__position_type="c-jus")
-                            | Q(positions__position_type="ass-jus")
-                            | Q(positions__position_type="ret-act-jus")
-                        )
-                        & (
-                            Q(positions__date_termination__isnull=True)
-                            & Q(positions__date_retirement__isnull=True)
-                        )
-                    ),
-                    positions__court_id="me",
-                    is_alias_of=None,
-                )
-                .annotate(
-                    custom_order=Case(
-                        When(positions__position_type="c-jus", then=Value(1)),
-                        When(
-                            positions__position_type="ass-jus", then=Value(2)
-                        ),
-                        When(
-                            positions__position_type="ret-act-jus",
-                            then=Value(3),
-                        ),
-                        output_field=IntegerField(),
-                    )
-                )
-                .order_by("custom_order", "positions__date_start")
-            )
-        else:
-            q_judges = Person.objects.filter(
-                positions__court_id=self.pk, is_alias_of=None
-            ).order_by("name_first")
+    def add_author_field(self, required=True):
+        """Add author field to form
 
-        for field_name in [
-            "lead_author",
-            "second_judge",
-            "third_judge",
-            "panel",
-        ]:
-            self.fields[field_name].queryset = q_judges  # type: ignore[attr-defined]
-            self.fields[field_name].label_from_instance = self.person_label  # type: ignore[attr-defined]
+        :param required: field is required or not
+        :return: None
+        """
+        lead_author = forms.ModelChoiceField(
+            queryset=Person.objects.none(),
+            required=required,
```

Comment:
This cant be required - at least not for all situations.  

we may not have many of the appellate judges in our system.